packages/bff/package.json

```diff
@@ -14,8 +14,9 @@
     "typeorm:run": "typeorm-ts-node-commonjs migration:run -- -d src/data-source.ts",
     "typeorm:revert": "npm run typeorm migration:revert",
     "dev": "concurrently \"npx tsc --watch\" \"nodemon --watch 'src/**/*.ts'\"",
+    "docker": "concurrently \"npx tsc --watch\" \"nodemon --watch '/app/packages/bff/src/**/*.ts'\"",
     "build:docker": "docker build -f ./Dockerfile -t bff ../..",
-    "run:docker": "docker run -it -p 80:80 bff"
+    "run:docker": "docker run -e IS_LOCAL=true -it -p 80:80 bff"
```

Comment:
Why do we need `IS_LOCAL`?

Reply:
To specify that we are in local development instead of in azure

Reply:
But, wouldn't it be better if the code was the same? This feels like we're creating "works on my machine" solutions.

Reply:
To my understand, `INITIALIZE APPLICATION INSIGHTS` is only to be enabled in Azure, right? And the check for this is `!isLocal`? 

Could we rather have a flag called `enable_application_insights` to be run in Azure env? And default off? 

IS_LOCAL is a bit vague to begin with, and certainly doesn't say anything about what it does.

Reply:
And as far as I can see, even this might not be neccessary since there is a check for `APPLICATIONINSIGHTS_CONNECTION_STRING` before enabling it.

Reply:
> To my understanding, `INITIALIZE APPLICATION INSIGHTS` is only to be enabled in Azure, right? And the check for this is `!isLocal`?
> 
> Could we rather have a flag called `enable_application_insights` to be run in Azure env? And default off?
> 
> IS_LOCAL is a bit vague to begin with, and certainly doesn't say anything about what it does.

It is used for more than just App Insights
![CleanShot 2024-02-26 at 10 04 33@2x](https://github.com/digdir/dialogporten-frontend/assets/144693023/6e6575a9-0f54-4406-910c-3ca6840681f9)

Reply:
then there should be a separate key for this, or we should consider https locally.
the point is still to deattach business logic from the config settings

Reply:
Can we create a backlog task for this and take it in its own PR?

Reply:
Yes!